wordpress-mobile/WordPress-Android
# Provide installable APK for every Pull Request

This PR adds a CircleCI job thats builds an APK which can be used to easily test changes on that PR.

A comment is added to the PR by Peril with a link to download the APK (see https://github.com/Automattic/peril-settings/pull/14) and the comment below.

In order for the app to behave just as the production app would (i.e. Google login, WP.com works), I have changed how credentials are included in the app. You can see how this works in https://github.com/wordpress-mobile/release-toolkit/pull/69 and https://github.com/wordpress-mobile/release-toolkit/pull/71.

The APK is built with `assembleVanillaRelease` but the `versionName` is changed to include the PR and build number, something like `WordPress-pr-10404-build-19775.apk`.

To test:

1. Download and install the APK linked below. Do a basic smoke test to make sure it works as expected.
2. Re-run the CircleCI build and verify that the download link is updated to the new APK.

Update release notes:
